Example/android/app/build.gradle

```
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Technically, there should be a newline after the comment
```suggestion
 */

```